Content.Shared/Atmos/Atmospherics.cs

```diff
@@ -273,6 +273,125 @@ static Atmospherics()
+
+        /// <summary>
+        ///     Standard Effect on PowerMixRatio
+        /// </summary>
+        public const float  TritiumPowerMixRatio = 1f;
+
+
+
+        // Higher == Bigger heat and waste penalty from having the crystal surrounded by this gas. Negative numbers reduce penalty.
+        /// <summary>
+        ///     if you pump the rock full of plasma its gonna melt, simple as
+        /// </summary>
+        public const float PlasmaHeatPenalty = 15f;
+
+        /// <summary>
+        ///     Oxy just makes it a little hot
+        /// </summary>
+        public const float OxygenHeatPenalty = 1f;
+
+        /// <summary>
+        ///     spice gas for the spice rock
+        /// </summary>
+        public const float TritiumHeatPenalty = 10f;
+
+        /// <summary>
+        ///     honestly have no idea why its 0.1 and not 0
+        /// </summary>
+        public const float CO2HeatPenalty = 0.1f;
+
+        /// <summary>
+        ///     keeps the spice rock cool
+        /// </summary>
+        public const float NitrogenHeatPenalty = -1.5f;
+
+        /// <summary>
+        ///     This'll get made slowly over time, I want my spice rock spicy god damnit
+        /// </summary>
+        public const float WaterHeatPenalty = 12f;
+
+
+
+        //All of these get divided by 10-bzcomp * 5 before having 1 added and being multiplied with power to determine rads
+        //Keep the negative values here above -10 and we won't get negative rads
+        //Higher == Bigger bonus to power generation.
+        /// <summary>
+        ///
+        /// </summary>
+        public const float OxygenTransmitModifier = 1.5f;
+
+        /// <summary>
+        ///     nitrogen is boring
+        /// </summary>
+        public const float NitrogenTransmitModifier = 0f;
+
+        /// <summary>
+        ///     co2 puts the sm to sleep
+        /// </summary>
+        public const float CO2TransmitModifier = 0f;
+
+        /// <summary>
+        ///     The funny gas really gets it going
+        /// </summary>
+        public const float PlasmaTransmitModifier = 4f;
+
+        /// <summary>
+        ///     Water is only a little spicy
+        /// </summary>
+        public const float WaterTransmitModifier = 2f;
+
+        /// <summary>
+        ///     The Spice Gas makes the Spice rock spicy
+        /// </summary>
+        public const float TritiumTransmitModifier = 30f;
+
+
+
+        /// <summary>
+        ///     Higher == less heat released during reaction
+        /// </summary>
+        public const float  ThermalReleaseModifier = 5f;
+
+        /// <summary>
+        ///     Higher == less plasma released by reaction
+        /// </summary>
+        public const float  PlasmaReleaseModifier = 750f;
+
+        /// <summary>
+        ///     Higher == less oxygen released at high temperature/power
+        /// </summary>
+        public const float  OxygenReleaseModifier = 325f;
+
+        #endregion
```

Comment:
This shouldn't be in here but in the `SupermatterComponent`. *See my other comment about making it a map*.